electora_py/api.py

```diff
@@ -17,7 +39,25 @@
 )
 
 
-def fetch_votes():
+def _fetch_vote_transactions(election_id: str, endpoint: Optional[str] = _ARWEAVE_GQL_ENDPOINT) -> Dict:
+    """Fetches all arweave transactions tagged for a particular ballot ID."""
+    transport = AIOHTTPTransport(url=endpoint)
+    client = Client(transport=transport, fetch_schema_from_transport=True)
+    variables_map = {
+        'transactions': {
+            'tags': {'tagName': _ELECTORA_ARWEAVE_TAG, 'electionId': election_id}
+        }
+    }
+    result = client.execute(
+        gql(_GET_ELECTION_VOTES_QUERY_TEMPLATE),
+        variable_values=variables_map
+    )
+    return result
+
+
+def fetch_votes(election_id: str, endpoint: Optional[str] = _ARWEAVE_GQL_ENDPOINT):
+    transactions = _fetch_vote_transactions(election_id=election_id, endpoint=endpoint)
+    print(transactions)
```

Comment:
Let's return these from here and add a cli command to do the print

Reply:
The `print` and `pass` here are meant to indicate the this is still stub functionality (the transaction results need further processing before we actually have _votes_ in scope).  Do you have a preferred way to indicate a stub?

Reply:
I'd raise a `NotImplementedError`

Reply:
✔️